src/snovault/elasticsearch/indexer.py

```diff
@@ -387,7 +385,6 @@ def _load_indexing(request, session, connection, indexer_state):
             else:
                 status = request.registry[ELASTIC_SEARCH].get(
                     index=request.registry.settings['snovault.elasticsearch.index'],
-                    doc_type='meta',
```

Comment:
Where does this get stored now? The `doc_type` concept is deprecated?

Reply:
yes, deprecated. Index is the doc_type for us now.